### DISCUSSION TITLE
duplicate labels in tabs

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#4421] [Feedback]
admittedly minor thing but it seems redundant to have another row that just says "Session" right under tab labelled "Session". This occurs in the other tabs as well (Variables, Help, etc...). Removing this extra space would help smaller laptop displays maximize plot real estate.

![image](https://github.com/user-attachments/assets/fe0ad858-0a40-4e2a-b4e3-b3642fce3e78)

**Comment by juliasilge** [comment]
Hmmmm, somehow you have dragged your Session tab into... your Session tab? Not totally sure how that happened but it _is_ possible to get into weird layout situations since so many components are draggable and customizable.

- As a workaround to get yourself back into a happy UI state, check out our layout presets:

![Screenshot 2024-08-21 at 9 57 23 AM](https://github.com/user-attachments/assets/4bf71562-e4b0-4e0c-9fa5-95ec019cbb40)

The "Stacked Layout" is most similar to RStudio, and if you like a lot of screen real estate for your code editor, check out "Side-By-Side Layout".

- If you can share a reproducible set of steps that get you into this weird state, that would be helpful because we may want to guard against getting into this state.

**Comment by brndngrhm** [comment]
weird - it was because i set the activity bar location to "Top" in my settings.json (`"workbench.activityBar.location": "top"`) and commenting that out removes the duplicate labels. i do prefer the activity bar location on top but the duplicate labels aren't great either, so i guess this is morphing into more of an enhancement request to allow activity bar location on top in a way that prevents those duplicate labels.

**Reply by juliasilge** [reply]
Ah yep, thank you! I have opened https://github.com/posit-dev/positron/issues/4427 to track and prioritize this work.
